home/templates/home/default_page.html

```diff
@@ -64,6 +64,50 @@
             color: #FFC107;
         }
 
+        footer {
+            position: absolute;
+            display: flex;
+            bottom: 0;
```

Comment:
When using absolute, how does it look when the page has enough information to scroll?

Maybe we should rather remove the absolute, and make the `.textBox` have flex-grow: 1, such that the navbar is always at the bottom of the screen

Reply:
You're probably right, i'll look more into what absolute does, and try out your solution :)